Makefile

```diff
@@ -0,0 +1,27 @@
+.PHONY: start start-debug stop clean
+
+# $SUDO is an optional shell variable which can be set to "sudo" if needed
+# (for example, on an ec2 instance)
+
+start:
+	@echo "--- Running 'docker-compose up --build -d'..."
+	GIT_HASH=`git log --pretty="%h" -n 1` ${SUDO} docker-compose up --build -d
+
+start-debug:
+	@echo "--- Running 'docker-compose --log-level DEBUG --verbose up --build -d'..."
+	GIT_HASH=`git log --pretty="%h" -n 1` ${SUDO} docker-compose --log-level DEBUG --verbose up --build -d
+
+stop:
+	@echo "--- Running 'docker-compose kill/rm/prune'..."
+	${SUDO} docker-compose kill
+	${SUDO} docker-compose rm -f
+	${SUDO} docker system prune --volumes -f
```

Comment:
This command is destructive and would wipe out all of my project data from other projects on my computer

Reply:
If used in production, I'm pretty sure this would wipe out all the data when `stop` is issued. That could be a nasty surprise!

Reply:
Good point. I will test.

Reply:
This is the docker equivalent of `sudo rm -rf /`
If this ran on my server it would destroy every webpage and database for all my clients.